src/contractTest/java/uk/gov/hmcts/reform/cosapi/clients/ccd/CaseManagementConsumerTest.java

```diff
@@ -0,0 +1,82 @@
+package uk.gov.hmcts.reform.cosapi.clients.ccd;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.dsl.PactDslJsonBody;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactConsumerTestExt;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import au.com.dius.pact.core.model.annotations.PactFolder;
+import java.io.IOException;
+import org.apache.http.HttpResponse;
+import org.apache.http.HttpStatus;
+import org.json.JSONException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.apache.http.client.fluent.Request;
+
+
+@ExtendWith(PactConsumerTestExt.class)
+@ExtendWith(SpringExtension.class)
+@PactTestFor(providerName = "ccd_submitForCitizen_api", port = "5001")
+@PactFolder("pacts")
+@SpringBootTest({  "ccd_submitForCitizen_api:http://localhost:5001"})
+public class CaseManagementConsumerTest {
+
+    private static final String SERVICE_AUTHORIZATION_HEADER = "ServiceAuthorization";
+    private static final String someServiceAuthToken = "someServiceAuthToken";
```

Comment:
these four variables along with consumer ="fis-cos-api" can be created as constants as these are used in most of the test classes